staging/src/k8s.io/apimachinery/pkg/labels/selector.go

```diff
@@ -299,7 +298,8 @@ func (r *Requirement) String() string {
 	if len(r.strValues) == 1 {
 		buffer.WriteString(r.strValues[0])
 	} else { // only > 1 since == 0 prohibited by NewRequirement
-		buffer.WriteString(strings.Join(r.strValues, ","))
+		// move sorting behavior from NewRequirement() to here
```

Comment:
Please make the comment describe why this is the right place, not where you moved it from.

Reply:
done.